### ISSUE TITLE
Dangerous mode with js cra crashes with: array expected for paths, specify

### ISSUE THREAD

**Issue by oklas** [oklas/react-app-rewire-alias#41]
Hello, I'm having the same issue. Updating the import path seems to break the plugins:
```
const {
  CracoAliasPlugin,
  configPaths,
} = require("react-app-rewire-alias/lib/aliasDangerous");

const aliasMap = configPaths("./jsconfig.paths.json");

module.exports = {
  plugins: [
    {
      plugin: CracoAliasPlugin,
      options: { alias: aliasMap },
    },
  ],
};
```
Result in:
```
Error:
      react-app-rewire-alias:configPaths: array expected for paths, specify ./jsconfig.paths.json instead of D:\Administration\administration-ui\jsconfig.json for configPaths()
    at configPathsRaw (D:\Administration\administration-ui\node_modules\react-app-rewire-alias\src\index.js:110:11)
    at expandPluginsTsChecker (D:\Administration\administration-ui\node_modules\react-app-rewire-alias\src\aliasDangerous.js:81:21)
    at D:\Administration\administration-ui\node_modules\react-app-rewire-alias\src\aliasDangerous.js:126:5
    at Object.overrideWebpackConfig (D:\Administration\administration-ui\node_modules\react-app-rewire-alias\src\aliasDangerous.js:133:62)
    at overrideWebpack (D:\Administration\administration-ui\node_modules\@craco\craco\lib\features\plugins.js:42:40)
    at D:\Administration\administration-ui\node_modules\@craco\craco\lib\features\plugins.js:64:29
    at Array.forEach (<anonymous>)
    at applyWebpackConfigPlugins (D:\Administration\administration-ui\node_modules\@craco\craco\lib\features\plugins.js:63:29)
    at mergeWebpackConfig (D:\Administration\administration-ui\node_modules\@craco\craco\lib\features\webpack\merge-webpack-config.js:110:30)
    at overrideWebpackDev (D:\Administration\administration-ui\node_modules\@craco\craco\lib\features\webpack\override.js:11:36)
error Command failed with exit code 1.
```

---

Using `require("react-app-rewire-alias");` does give me the `aliasDangerous` warning as expected: 
```
$ craco start
alias 'GameConfigs' is outside of root - supported only by aliasDangerous
https://github.com/oklas/react-app-rewire-alias#outside-of-root
```

_Originally posted by @AVAVT in https://github.com/oklas/react-app-rewire-alias/issues/25#issuecomment-956122063_

**Comment by oklas** [comment]
@oklas Hi, so I will describe my situation in steps:

1. I create a CRA app (the `npx create-react-app my-app` thing). Adding `craco` and `react-app-rewire-alias`. With the following configs:

`jsconfig.paths.json`:

```
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "GameConfigs/*": ["../GameConfigs/*"]
    }
  }
}
```

`craco.config.js`

```
const {
  CracoAliasPlugin,
  configPaths,
} = require("react-app-rewire-alias/lib/aliasDangerous");

const aliasMap = configPaths("./jsconfig.paths.json");

module.exports = {
  plugins: [
    {
      plugin: CracoAliasPlugin,
      options: { alias: aliasMap },
    },
  ],
  style: {
    postcss: {
      plugins: [require("tailwindcss"), require("autoprefixer")],
    },
  },
};
```

`jsconfig.json`

```
{
  "extends": "./jsconfig.paths.json",
  "compilerOptions": {
  }
}
```

2. The above configs give me the errors I mentioned in previous comment.

============

3. After trying many things to no avail. I remember a situation I had before where the libs don't work in js CRA but works with ts CRA, so I create another CRA app with typescript template (the `npx create-react-app my-app --template typescript` thing)

After that I use the exact same 3 files above, changing all `js` into `ts`.

And it worked.

_Originally posted by @AVAVT in https://github.com/oklas/react-app-rewire-alias/issues/25#issuecomment-963974731_

**Comment by oklas** [comment]
Released v1.1.5
